pkg/services/devices.go

```diff
@@ -629,8 +629,17 @@ func (s *DeviceService) GetDevicesView(limit int, offset int, tx *gorm.DB) (*mod
 	// create a map of unique image id's. We dont want to look of a given image id more than once.
 	setOfImages := make(map[uint]*neededImageInfo)
 	for _, devices := range storedDevices {
+		var status = ""
+		if devices.UpdateTransaction != nil && len(*devices.UpdateTransaction) > 0 {
+			status = (*devices.UpdateTransaction)[len(*devices.UpdateTransaction)-1].Status
+			if status == models.UpdateStatusBuilding {
+				status = models.DeviceViewStatusUpdating
+			} else {
+				status = models.DeviceViewStatusRunning
+			}
+		}
```

Comment:
```suggestion
		var status = models.DeviceViewStatusRunning
		if devices.UpdateTransaction != nil && len(*devices.UpdateTransaction) > 0 {
			status = (*devices.UpdateTransaction)[len(*devices.UpdateTransaction)-1].Status
			if status == models.UpdateStatusBuilding {
				status = models.DeviceViewStatusUpdating
			}
		}
```

WDYT with we go with that?

Reply:
Seems ok, 

Ill add now
Tyvm!